### DISCUSSION TITLE
SOK V4 bluetooth only BMS (ABC-BMS app)

### DISCUSSION THREAD

**Discussion by filemoon** [mr-manuel/venus-os_dbus-serialbattery#62] [New BMS requests]
Hi,

Can you add SOK (v4) bluetooth BMS into supported BMS list?

**Comment by mr-manuel** [comment]
Hello,
to add a new BMS there are a few possibilities.
- If enough users demand for it and I get a BMS to test, I can add it for free.
- If there are not enough users demanding for it you could bug bounty the development to add this BMS, which are at least 12 hours of work at a rate of 100 €/h. [Make a donation](https://www.paypal.com/donate/?hosted_button_id=3NEVZBDM5KABW) and insert the BMS as note. I will update the state here how much was donated for reaching this feature.
- Else you can add it yourself by opening a PR.
